packages/back-end/src/integrations/SqlIntegration.ts

```diff
@@ -750,16 +752,24 @@ export default abstract class SqlIntegration
     query: string,
     templateVariables?: TemplateVariables
   ): string {
-    return this.getTestQuery(query, templateVariables, 1);
+    return this.getTestQuery(
+      query,
+      templateVariables,
+      DEFAULT_TEST_QUERY_DAYS,
+      1
+    );
   }
 
   getTestQuery(
     query: string,
     templateVariables?: TemplateVariables,
+    testDays?: number,
     limit: number = 5
```

Comment:
Nit: It's not very developper friendly to have multiple optional arguments without switching to a single hash/record parameter. Otherwise, you have to pass `testDays` explicitly to be able to pass `limit` etc. Another benefit of a single hash/record parameter is to have named arguments.

Reply:
Moved to a params object.